src/overlays/actors/ovl_En_Rsn/z_en_rsn.c

```diff
@@ -61,14 +68,14 @@ void EnRsn_Update(Actor* thisx, PlayState* play) {
     this->actionFunc(this, play);
     Actor_MoveWithGravity(&this->actor);
     SkelAnime_Update(&this->skelAnime);
-    Actor_TrackPlayer(play, &this->actor, &this->unk1D8, &this->unk1DE, this->actor.focus.pos);
+    Actor_TrackPlayer(play, &this->actor, &this->headRot, &this->torsoRot, this->actor.focus.pos);
 }
 
 s32 EnRsn_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
     EnRsn* this = THIS;
 
     if (limbIndex == BOMB_SHOPKEEPER_LIMB_RIGHT_HAND) {
-        Matrix_RotateXS(this->unk1D8.y, MTXMODE_APPLY);
+        Matrix_RotateXS(this->headRot.y, MTXMODE_APPLY);
```

Comment:
This rot obtained from `Actor_TrackPlayer` being applied to a right-hand limb seems sus, worth double checking this

Reply:
Pretty sure this actor is only used in the credits. Someone would either need to breakout of the credits, or spawn him else where to check.

Interesting that EnSob1 has the proper limb.
```c
s32 EnSob1_BombShopkeeper_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
                                           Actor* thisx) {
    EnSob1* this = THIS;

    if (limbIndex == BOMB_SHOPKEEPER_LIMB_HEAD) {
        Matrix_RotateXS(this->headRot, MTXMODE_APPLY);
    }
    return false;
}
```